### PR TITLE
Upgrading Dock SDK + ReactNative to 0.70

### PR DESCRIPTION
**Pull request by maycon-mello**

### CODE REVIEW

File: android/app/build.gradle [hunk 1]
```diff
@@ -142,13 +144,72 @@ android {
         versionCode 254
         versionName "0.2.5"
         missingDimensionStrategy 'react-native-camera', 'general'
+        buildConfigField "boolean", "IS_NEW_ARCHITECTURE_ENABLED", isNewArchitectureEnabled().toString()
+        if (isNewArchitectureEnabled()) {
```
Comment (mike-parkhill) [inline android/app/build.gradle:148]: are we going to need to support the old architecture going forward?

File: android/app/build.gradle [hunk 2]
```diff
@@ -142,13 +144,72 @@ android {
...
+            if (!enableSeparateBuildPerCPUArchitecture) {
+                ndk {
+                    abiFilters (*reactNativeArchitectures())
+                }
+            }
+        }
     }
+
+    if (isNewArchitectureEnabled()) {
+        // We configure the NDK build only if you decide to opt-in for the New Architecture.
```
Comment (mike-parkhill) [inline android/app/build.gradle:168]: could these blocks be pulled into helper functions to reduce nesting and maybe make it easier to read?

File: android/app/src/main/java/com/dockapp/MainApplication.java [hunk 3]
```diff
@@ -4,13 +4,15 @@
 import android.content.Context;
 import com.facebook.react.PackageList;
 import com.facebook.react.ReactApplication;
-import com.reactlibrarylizhming.ScreenCaptureSecurePackage;
+// import com.reactlibrarylizhming.ScreenCaptureSecurePackage;
```
Comment (mike-parkhill) [inline android/app/src/main/java/com/dockapp/MainApplication.java:7]: can we delete this?

File: android/app/build.gradle (hunk 1 above)
Comment (maycon-mello) [reply android/app/build.gradle:148]: yes, this is the default for react-native 0.70 native code

File: metro.config.js [hunk 4]
```diff
@@ -36,6 +36,7 @@ module.exports = (async () => {
         vm: require.resolve('vm-browserify'),
         src: path.resolve(__dirname, './src'),
         realm: require.resolve('realm'),
+        // 'react-native-tcp': require.resolve('react-native-tcp-socket'),
```
Comment (mike-parkhill) [inline metro.config.js:39]: can we delete this?

File: android/app/src/main/java/com/dockapp/MainApplication.java (hunk 3 above)
Comment (mike-parkhill) [reply android/app/src/main/java/com/dockapp/MainApplication.java:7]: I think we still need that comment and maybe a follow up ticket in Jira for this

Comment (maycon-mello) [reply android/app/src/main/java/com/dockapp/MainApplication.java:7]: @mike-parkhill https://dock-team.atlassian.net/browse/DCK-2277 i've created a jira ticket

**Review by mike-parkhill — APPROVED** [review]

**Pull request merged**